lib/env-bundler.js

```diff
@@ -282,9 +340,9 @@ const collectQuestions = [
       { name: 'Microservices', value: 'plugin' },
       { name: 'Content Templates', value: 'contentModel' },
       { name: 'Content Types', value: 'contentType' },
-      { name: 'Resources', value: 'resource' },
       { name: 'Groups', value: 'group' },
       { name: 'Categories', value: 'category' },
+      { name: 'File Browser', value: 'resource' },
```

Comment:
@ffleandro Assets?

Reply:
It's the name of the descriptor/folder inside the bundle. In the case of the File Tree, everything goes into a `resources` folder.